### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by leewang0**
## Description of the change

v2.0.0 Updates
* new models per the latest Merge API spec including `Group` object and `PayGroup`s
* nuget configs
* github build hook for nuget

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests, or the description above explains how testing was performed

### Code review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or gif attached. Download "Kap" from the Mac App store for easy gif screen capture.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+
+      - name: Build & package project
+        run: |
+          dotnet pack -c Release -o out
+
+      - name: Push to GitHub package registry
+        run: dotnet nuget push ./out/*.nupkg --source ${{ env.NUGET_REPOSITORY_URL }} --api-key ${{ secrets.NUGET_API_KEY }} --skip-duplicate --no-symbols true
```
Comment (leewang0) [inline .github/workflows/publish.yml:30]: nuget api token was created and github secret was added to support this

**Review by eltonkl — APPROVED** [review]

File: docs/DataPassthroughRequest.md [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
 ------------ | ------------- | ------------- | -------------
-**Method** | **string** |  | 
+**Method** | **MethodEnum** |  | 
 **Path** | **string** |  | 
 **BaseUrlOverride** | **string** |  | [optional] 
 **Data** | **string** |  | [optional] 
+**MultipartFormData** | [**List&lt;MultipartFormFieldRequest&gt;**](MultipartFormFieldRequest.md) | Pass an array of &#x60;MultipartFormField&#x60; objects in here instead of using the &#x60;data&#x60; param if &#x60;request_format&#x60; is set to &#x60;MULTIPART&#x60;. | [optional] 
 **Headers** | **Dictionary&lt;string, Object&gt;** |  | [optional] 
-**RequestFormat** | **string** |  | [optional] 
+**RequestFormat** | **RequestFormatEnum** |  | [optional]
```
Comment (dan-merge) [inline docs/DataPassthroughRequest.md:14]: I forgot, can c# handle enums?

Comment (leewang0) [reply docs/DataPassthroughRequest.md:14]: Oh good point, rip, this needs the enum + string treatment.

**Comment by leewang0** [comment]
Closing in favor of redoing the whole sdk in the future

**Pull request closed without merging**